third_party/terraform/utils/iam_bigquery_dataset.go

```diff
@@ -190,6 +190,9 @@ func policyToAccess(policy *cloudresourcemanager.Policy) ([]map[string]interface
 // Dataset access uses different member types to identify groups, domains, etc.
 // these types are used as keys in the access JSON payload
 func iamMemberToAccess(member string) (string, string, error) {
+	// Remove any occurrence of "deleted:"
+	member = strings.TrimPrefix(member, "deleted:")
```

Comment:
I don't think this is what we want here. This will remove `deleted:` which will make Terraform believe that the non-deleted member has access when they shouldn't, and cause subsequent updates to try to add back permission to a member that may no longer exist

Can we check for `deleted:` and then change the switch to read from the correct spot, then return the original member so we don't modify members in here?

Reply:
Tried to modify the switch block and send the full member string as the email (`deleted:type:email`) but got an error from the API.
Instead, I modified the logic to just skip any members with `deleted:` prefix

Reply:
do you receive `Error: Provider produced inconsistent result after apply`?

Reply:
@edwardmedia for what configuration?

Reply:
good with me now